src/insertItem.ts

```diff
@@ -425,11 +403,11 @@ export class InsertItem {
                 prepend = `,\r\n\t\t`;
             }
         }
-        const resource = await this.ui.showQuickPick(getResourceSnippets(), { placeHolder: 'What resource do you want to insert?' });
+        const resource = await this.ui.showQuickPick(await getResourceSnippets(), { placeHolder: 'What resource do you want to insert?' });
         await this.insertText(textEditor, index, prepend);
         let newCursorPosition = this.getCursorPositionForInsertResource(textEditor, index, prepend);
         textEditor.selection = new vscode.Selection(newCursorPosition, newCursorPosition);
-        await commands.executeCommand('editor.action.insertSnippet', { name: resource.label });
+        await commands.executeCommand('editor.action.insertSnippet', { snippet: resource.data.insertText });
```

Comment:
Now we we insert the snippet by text rather than name since vscode no longer knows about them.